code/modules/mob/living/simple_animal/borer/borer.dm

```diff
@@ -0,0 +1,799 @@
+/mob/living/captive_brain
+	name = "host brain"
+	real_name = "host brain"
+
+/mob/living/captive_brain/say(var/message)
+
+	if (src.client)
+		if(client.prefs.muted & MUTE_IC)
+			src << "\red You cannot speak in IC (muted)."
+			return
+		if (src.client.handle_spam_prevention(message,MUTE_IC))
+			return
+
+	if(istype(src.loc,/mob/living/simple_animal/borer))
+
+		message = sanitize(message)
```

Comment:
will sanitizing the message cause double sanitation errors if it goes via say_dead instead?